src/ContinuousEnergyNeutron.tex

```diff
@@ -110,16 +110,19 @@ \subsubsection{\textsf{ESZ} Block}\label{sec:ESZBlock}
 \end{ThreePartTable}
 
 \subsubsection{\textsf{NU} Block}\label{sec:NUBlock}
-There are three possibilities for the \Block{NU}:
+There are two possibilities for the \Block{NU}:
 \begin{enumerate}
   \item {\bfseries\sffamily No NU Block.} (\jxs{2}=0)
-  \item {\bfseries\sffamily Either prompt or total \nubar\ is given (but not both).} (\jxs{2} > 0) \\
-    The \aceArray{NU} array begins at location \xss{KNU} where \var{KNU}=\jxs{2} + 1.
-  \item {\bfseries\sffamily Both prompt and total \nubar\ are given.} (\jxs{2} < 0).
-    \begin{itemize}
-      \item The prompt \aceArray{NU} array begins at \xss{KNU} where \var{KNU}=\jxs{2} + 1. 
-      \item The total \aceArray{NU} array begins at \xss{KNU} where {\sffamily \var{KNU} = \jxs{2} + ABS(\xss{\jxs{2}})+1}.
-    \end{itemize}
+  \item {\bfseries\sffamily NU Block exists.} (\jxs{2} > 0)
+  \begin{itemize}
+	\item {\bfseries\sffamily Prompt or total \nubar\ is given (but not both).} (\xss{\jxs{2}} > 0) \\
+	The \aceArray{NU} array begins at location \xss{KNU} where \var{KNU}=\jxs{2} + 1.
+	\item {\bfseries\sffamily Both prompt and total \nubar\ are given.} (\xss{\jxs{2}} < 0)
```

Comment:
In this case, ```\xss{\jxs{2}}``` is a locator for the total nu array. This value is not present when either prompt or total is given. The quoted values for ```\var{KNU}``` given below are therefore correct.

We should maybe rewrite this piece of the ACE specification to better reflect the presence of the locator.

Reply:
This is a good point, this locator should be emphasized.